fcrepo-indexing-solr/src/main/java/org/fcrepo/camel/indexing/solr/SolrRouter.java

```diff
@@ -43,6 +43,8 @@
     private static final String hasIndexingTransformation =
         "/rdf:RDF/rdf:Description/indexing:hasIndexingTransformation/text()";
 
+    private static final String RESOURCE_DELETION = "http://fedora.info/definitions/v4/event#ResourceDeletion";
```

Comment:
As long as you are keeping track of the required refactoring of this following the 4.7.0 Fedora release, it is fine by me.
Maybe a comment would help with that:
```
// TODO: Refactor this into...
```
..for example.